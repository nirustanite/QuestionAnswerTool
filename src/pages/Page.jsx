import React from "react";
import { Container } from 'semantic-ui-react';
import styled from 'styled-components';
import Nav from 'Components/Nav/Nav';

const StyledContainer = styled(Container)`
    padding-top: 5em;
    padding-bottom: 5em;
`;

const Page = ({ children }) => {
    return (
        <React.Fragment>
            <Nav />
            <StyledContainer>{children}</StyledContainer>
        </React.Fragment>
    );
};

export default Page;
