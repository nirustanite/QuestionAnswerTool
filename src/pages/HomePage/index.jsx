import React from 'react';
import Page from '../Page';

const HomePage = () => {
    return(
        <Page>
            Welcome to QA Tool
        </Page>
    );
};

export default HomePage;