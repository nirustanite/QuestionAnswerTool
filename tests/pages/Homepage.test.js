import React from 'react';
import { shallow } from 'enzyme';
import HomePage from 'Pages/HomePage';
import Page from 'Pages/Page';

describe('HomePage', () => {

    it('renders without error', () => {
		const wrapper = shallow(<HomePage />);
		expect(wrapper.length).toEqual(1);
	});

    it('should render the HomePage Component correctly', () => {   
        const wrapper = shallow(<Page>Welcome</Page>);
        expect(wrapper).toMatchSnapshot();
    });

});