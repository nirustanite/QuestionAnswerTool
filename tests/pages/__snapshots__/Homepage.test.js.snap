// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should render the HomePage Component correctly 1`] = `ShallowWrapper {}`;
